string str
---
string data
uint32 poseQueueSize